### ISSUE TITLE
what is difference between trrain.py and pissa_finetuning.py

### ISSUE THREAD

**Issue by azuryl** [GraphPKU/PiSSA#3]
thank your great work
excuse me 
there train.py and https://github.com/fxmeng/peft/blob/main/examples/pissa_finetuning/pissa_finetuning.py
 I want to fine tuning a llama-7B model, which file I would use?

Thank you

**Comment by fxmeng** [comment]
Thank you for your interest in PiSSA. 
We recommend using `train.py` for training. 
`train.py` is the code used to compare full parameter fine-tuning, LoRA, and PiSSA, which can replicate the results presented in the paper. 
On the other hand, `pissa_finetuning.py` is designed to demonstrate various advanced uses of PiSSA. 
The main difference between the two is that `train.py` calculates only the loss of the response during training, which has been proven to yield better results in QLoRA experiments. In contrast, `pissa_finetuning.py` does not employ this technique for ease of understanding.
Best regard.
